CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -265,28 +281,62 @@ local function createImageButtonWithText(name, position, image, imageDown, text,
+	else
+		smooth.ImageColor3 = backgroundColor3
+		smooth.ImageTransparency = backgroundTransparency
+	end
+end
+
+local function getFrameBackground(frame)
+	local smooth = frame:FindFirstChild("Smooth3D")
+	if not smooth then
+		return frame.BackgroundColor3, frame.BackgroundTransparency
+	else
+		return smooth.ImageColor3, smooth.ImageTransparency
+	end
+end
+
 --[[ Begin Gui Creation ]]--
 local PurchaseDialog = isTenFootInterface and createFrame("PurchaseDialog", DIALOG_SIZE_TENFOOT, HIDE_POSITION_TENFOOT, 1, nil) or createFrame("PurchaseDialog", DIALOG_SIZE, HIDE_POSITION, 1, nil)
 PurchaseDialog.Visible = false
 PurchaseDialog.Parent = RobloxGui
 
-	local ContainerFrame = createFrame("ContainerFrame", UDim2.new(1, 0, 1, 0), nil, 1, nil)
+	local ContainerFrame = createFrame("ContainerFrame", UDim2.new(1, 0, 1, 0), nil, 0.5, Color3.new(31/255,31/255,31/255))
+	ContainerFrame.ZIndex = 8
 	ContainerFrame.Parent = PurchaseDialog
 
-		local ContainerImage = createImageLabel("ContainerImage", isTenFootInterface and BODY_SIZE_TENFOOT or BODY_SIZE, UDim2.new(0, 0, 0, 0), BG_IMAGE)
-		ContainerImage.ZIndex = 8
-		ContainerImage.Parent = ContainerFrame
+		local WindowTitle = createTextLabel("WindowTitle", UDim2.new(1, 0, 0, 40), UDim2.new(0, 0, 0, 0), Enum.Font.SourceSansBold, Enum.FontSize.Size24, "Confirm Purchase")
+		WindowTitle.Parent = ContainerFrame
+		WindowTitle.ZIndex = 9
 
-		local ItemPreviewImage = isTenFootInterface and createImageLabel("ItemPreviewImage", UDim2.new(0, 64*scaleFactor, 0, 64*scaleFactor), UDim2.new(0, 27*scaleFactor, 0, 20*scaleFactor), "") or createImageLabel("ItemPreviewImage", UDim2.new(0, 64, 0, 64), UDim2.new(0, 27, 0, 20), "")
+		local ColorStripe = createFrame("ColorStripe", UDim2.new(1, 0, 0, 2), nil, 0, Color3.new(0.01, 0.72, 0.34))
+		ColorStripe.Position = UDim2.new(0, 0, 0, 40)
+		ColorStripe.ZIndex = 9
+		ColorStripe.Parent = ContainerFrame
+
+		local ItemPreviewImage = isTenFootInterface and createImageLabel("ItemPreviewImage", UDim2.new(0, 64*scaleFactor, 0, 64*scaleFactor), UDim2.new(0, 27*scaleFactor, 0, 60*scaleFactor), "") or createImageLabel("ItemPreviewImage", UDim2.new(0, 64, 0, 64), UDim2.new(0, 27, 0, 60), "")
 		ItemPreviewImage.ZIndex = 9
 		ItemPreviewImage.Parent = ContainerFrame
 
-		local ItemDescriptionText = createTextLabel("ItemDescriptionText", isTenFootInterface and UDim2.new(0, 210*scaleFactor - 20, 0, 96*scaleFactor) or UDim2.new(0, 210, 0, 96), isTenFootInterface and UDim2.new(0, 110*scaleFactor, 0, 18*scaleFactor) or UDim2.new(0, 110, 0, 18),
-			Enum.Font.SourceSans, isTenFootInterface and Enum.FontSize.Size48 or Enum.FontSize.Size18, PURCHASE_MSG.PURCHASE)
+		local ItemDescriptionText = createTextLabel("ItemDescriptionText", isTenFootInterface and UDim2.new(0, 210*scaleFactor - 20, 0, 96*scaleFactor) or UDim2.new(0, 210, 0, 96), isTenFootInterface and UDim2.new(0, 110*scaleFactor, 0, 58*scaleFactor) or UDim2.new(0, 110, 0, 58),
```

Comment:
the whole point of scaleFactor is so we don't need to have different pixel values for 10 foot and regular. If it seems like we can't get away with using the same base values, we should remove scaleFactor because it doesn't really add anything otherwise.